ova/Vagrantfile

```diff
@@ -3,21 +3,24 @@
 
 Vagrant.configure("2") do |config|
 
-  config.vm.box = "centos/7"
+  config.vm.box_url = "https://packages-dev.wazuh.com/vms/ova/amazonlinux-2.box"
```

Comment:
Why are we using a -dev url for the image here? this is going to be merged in a stable branch

Reply:
And where is this new variable used? Why wasn't it used before?

Reply:
It is the definition that we give to Vagrant about which box to use, since it is a private box, it is passed through the URL from where the box should be downloaded.

It is in packages-dev because this box is not a Wazuh deliverable itself, the deliverable is the OVA, we use this box as a base to build the OVA.

The box is public anyway